src/swoop/api/workflows.py

```diff
@@ -0,0 +1,7 @@
+from fastapi import FastAPI
+import yaml
+
+
+def init_workflows_config(app: FastAPI) -> None:
+    """Initialize Workflow Config."""
+    app.state.workflows = yaml.full_load(open(app.state.settings.workflow_config_file))
```

Comment:
I think you want safe_load here? Or is there a reason you have to do the full load?

```suggestion
    app.state.workflows = yaml.safe_load(open(app.state.settings.workflow_config_file))
```